bokehjs/src/coffee/api/typings/models/plots.d.ts

```diff
@@ -3,6 +3,11 @@ declare namespace Bokeh {
         plot?: Plot;
     }
 
+    export interface PlotView extends View<Plot> {
+        model: Plot;
+        resize_width_height(use_width:boolean, use_height:boolean, maintain_ar?:boolean, width?:number, height?:number): void;
+    }
+
```

Comment:
@mattpap is this correct? Gulp does not error on it, but I am not very familiar with the working of these files.